DEBUG:root:Debug message
INFO:root:Info message
ERROR:root:Error message
CRITICAL:root:Critical message
INFO:root:Division Process Starting
ERROR:root:division by zero
Traceback (most recent call last):
  File "c:\Users\ANWAR\Desktop\python_course\class_12\test_logging.py", line 29, in <module>
    div = x/y
ZeroDivisionError: division by zero
INFO:root:Division Completed
INFO:root:all done
INFO:root:Info message
ERROR:root:Error message
CRITICAL:root:Critical message
INFO:root:Division Process Starting
ERROR:root:division by zero
Traceback (most recent call last):
  File "c:\Users\ANWAR\Desktop\python_course\class_12\test_logging.py", line 44, in <module>
    div = x/y
ZeroDivisionError: division by zero
INFO:root:Division Completed
INFO:root:all done
INFO:root:Info message
ERROR:root:Error message
CRITICAL:root:Critical message
INFO:root:Division Process Starting
INFO:root:Division Completed
INFO:root:all done
INFO:root:Info message
ERROR:root:Error message
CRITICAL:root:Critical message
INFO:root:Division Process Starting
ERROR:root:division by zero
Traceback (most recent call last):
  File "c:\Users\ANWAR\Desktop\python_course\class_12\test_logging.py", line 44, in <module>
    div = x/y
ZeroDivisionError: division by zero
INFO:root:Division Completed
INFO:root:all done
INFO:root:Info message
ERROR:root:Error message
CRITICAL:root:Critical message
INFO:root:Division Process Starting
ERROR:root:division by zero
Traceback (most recent call last):
  File "c:\Users\ANWAR\Desktop\python_course\class_12\test_logging.py", line 44, in <module>
    div = x/y
ZeroDivisionError: division by zero
INFO:root:Division Completed
INFO:root:all done
